src/display/canvas.js

```diff
@@ -37,6 +37,9 @@ const MIN_FONT_SIZE = 16;
 const MAX_FONT_SIZE = 100;
 const MAX_GROUP_SIZE = 4096;
 
+const CSS_PIXELS_PER_INCH = 96.0;
+const PDF_PIXELS_PER_INCH = 72.0;
```

Comment:
Given how these values are being used, it appears (as far as I can tell) to be basically the following viewer constant: https://github.com/mozilla/pdf.js/blob/5678c755624a4ab4ff7ebca8620ab1f317568633/web/ui_utils.js#L16

Can we perhaps move the existing `CSS_UNITS` constant into `display_utils.js` (and export it via `pdf.js`) and then re-use here too, instead of essentially duplicating this?

Reply:
Looking at the final patch, I've got one small suggestion for simplifying the new exports a little bit.
However, in the interest of getting this landed sooner rather than later, I'll merge it as-is and file a follow-up PR instead.